zipkin-storage/elasticsearch/src/test/java/zipkin2/elasticsearch/internal/client/HttpCallTest.java

```diff
@@ -270,6 +264,25 @@
         "Fielddata is disabled on text fields by default. Set fielddata=true on [spanName] in order to load fielddata in memory by uninverting the inverted index. Note that this can however use significant memory. Alternatively use a keyword field instead.");
   }
 
+  @Test void streamingContent() throws Exception {
```

Comment:
I love this test

Reply:
I don't - added checks to make sure `headers()` is asserted too so maybe now love-worthy :)